src/platform/plugins/shared/discover/public/context_awareness/profile_providers/observability/traces_document_profile/span_document_profile/profile.test.ts

```diff
@@ -16,12 +16,20 @@ import type { ContextWithProfileId } from '../../../../profile_service';
 import { OBSERVABILITY_ROOT_PROFILE_ID } from '../../consts';
 import type { ProfileProviderServices } from '../../../profile_provider_services';
 import { applicationMock } from '../__mocks__/application_mock';
+import { DEFAULT_ALLOWED_TRACES_BASE_PATTERNS } from '@kbn/discover-utils/src';
 
 describe('spanDocumentProfileProvider', () => {
-  const ROOT_CONTEXT: ContextWithProfileId<RootContext> = {
-    profileId: OBSERVABILITY_ROOT_PROFILE_ID,
-    solutionType: SolutionType.Observability,
+  const ROOT_CONTEXT = ({
```

Comment:
Maybe? this is now a function.
```suggestion
  const getRootContext = ({
```

Reply:
https://github.com/elastic/kibana/pull/215454/commits/e867669d37c023872192889688eb31e0af632453